native/src/sagas/watchFetchCities.ts

```diff
@@ -16,6 +17,7 @@ export function* fetchCities(dataContainer: DataContainer, action: FetchCitiesAc
     yield* put(insert)
   } catch (e) {
     console.error(e)
+    reportError(e)
```

Comment:
As I for my part don't like the console.errors in the unit tests much perhaps we could use the new reportError hook for this also and make it configurable?

I had a quick look, seems there are mor places where we have a console.error and the new reportError Hook at the same time.

Locally (developing) we don't need the Sentry report and on the other hand the console.errors aren't used for Sentry supervision.

Only an idea - and if we do it probalby a new task would make sense

Reply:
Sure, could you create a task? We could also suppress console errors in unit tests by replacing it with a mock I guess.